### PR TITLE
Performance measurement

### PR DESCRIPTION
**Pull request by lamphamsy**
Performance is measured using `s3blaster` tool. Hence, useless files will be removed.
It measures the three metrics for single S3 server:
- Lowest latency (in ms)
- Max number of operations/s: for single and multiple buckets
- Max throughput: for single and multiple buckets

Update package.json file
- Add s3blaster repo
- Update dependencies
- Add command to run perf test: `npm run perf`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,12 +51,16 @@
...
     "eslint-config-scality": "scality/Guidelines",
+    "generic-pool": "^2.4.2",
     "istanbul": "1.0.0-alpha.2",
     "istanbul-api": "1.0.0-alpha.13",
     "lolex": "^1.4.0",
     "mocha": "^2.3.3",
     "mocha-junit-reporter": "1.11.1",
-    "node-mocks-http": "^1.5.2"
+    "node-mocks-http": "^1.5.2",
+    "readline-sync": "^1.4.1",
```
Comment (michael-zapata) [inline package.json]: All the added devDependencies aside from `s3blaster` are not used.

File: package.json [hunk 2]
```diff
@@ -66,6 +71,7 @@
     "start": "node init.js && node index.js",
     "start_utapi": "node utapiServer.js",
     "test": "S3BACKEND=mem mocha --compilers js:babel-core/register --recursive tests/unit",
-    "unit_coverage": "mkdir -p coverage/unit/ && S3BACKEND=mem MOCHA_FILE=$CIRCLE_TEST_REPORTS/unit/unit.xml istanbul cover --dir coverage/unit _mocha -- --compilers js:babel-core/register --reporter mocha-junit-reporter --recursive tests/unit"
+    "unit_coverage": "mkdir -p coverage/unit/ && S3BACKEND=mem MOCHA_FILE=$CIRCLE_TEST_REPORTS/unit/unit.xml istanbul cover --dir coverage/unit _mocha -- --compilers js:babel-core/register --reporter mocha-junit-reporter --recursive tests/unit",
+    "perf": "mocha --compilers js:babel-core/register tests/performance/s3standard.js"
```
Comment (michael-zapata) [inline package.json]: Nit: Could we keep the list ordered?

File: package.json [hunk 3]
```diff
@@ -33,6 +33,7 @@
     "multilevel": "^7.3.0",
     "node-uuid": "^1.4.3",
     "ready-set-stream": "^1.0.0",
+    "s3blaster": "scality/s3blaster",
```
Comment (michael-zapata) [inline package.json]: This could be a devDep, right ?

**Comment by lamphamsy** [comment]
Updated package.json file

File: tests/performance/s3standard.js [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+    it('Put, get, then delete', done => {
+        params.output = `bkt${params.bucketsNb}_throughput`;
+        process.nextTick(runS3Blaster.start, params, done);
+    });
+});
+
+/*
+ * Clean databases
+ */
+describe('Clean databases of simulation', function fn() {
```
Comment (LaurenSpiegel) [inline tests/performance/s3standard.js]: Shouldn't we do a cleanup as an afterEach for each test?

Comment (lamphamsy) [inline tests/performance/s3standard.js]: Yes, it should be. I change `dontCleanDB = false` to cleanup after each test.

**Pull request merged**